crates/sui-types/src/quorum_driver_types.rs

```diff
@@ -111,7 +112,13 @@ pub struct QuorumDriverRequest {
 #[derive(Debug, Clone)]
 pub struct QuorumDriverResponse {
     pub effects_cert: VerifiedCertifiedTransactionEffects,
-    pub events: TransactionEvents,
+    // pub events: TransactionEvents,
+    pub events: Option<TransactionEvents>,
+    // Input objects will only be populated in the happy path
```

Comment:
Would be good to have a top level test that exercise the unhappy path

Reply:
we can probably add a test to check the case where the validator has already executed the txn, but we won't be able to add one that tests when objects have been pruned.

Reply:
> we can probably add a test to check the case where the validator has already executed the txn

Sounds good enough to me for the moment